### ISSUE TITLE
print() output not shown in logs for Swift actions

### ISSUE THREAD

**Issue by bjustin-ibm** [openwhisk/openwhisk#384]
For the following Swift action:
```swift
func main(args: [String:Any]) -> [String:Any] {
    print("Hey! Did this get logged?")
    return ["message": "No, it did not."]
}
```
I would expect to see "Hey! Did this get logged?" somewhere in the activation logs but, alas, I do not.

**Comment by csantanapr** [comment]
FYI: this affects both swift2 and swift3 runtimes

**Comment by psuter** [comment]
Should be fixed after #386 and tested explicitly, for both runtimes.

**Comment by csantanapr** [comment]
Yay viva the logs. !
